packages/material-ui/src/useAutocomplete/useAutocomplete.js

```diff
@@ -0,0 +1,4 @@
+export {
+  useAutocomplete as default,
+  createFilterOptions,
+} from '@material-ui/unstyled/AutocompleteUnstyled';
```

Comment:
Why not export * instead of listing each module?

Reply:
No reason, actually. With just one named export it doesn't make a difference.
But yeah, I'll change it in case anyone adds something new to useAutocomplete.js.

Reply:
Yes, this was my concern, that it could great bugs if we change the exports. If you could do a follow up, it would be perfect.

Reply:
I've corrected it already in this PR.